prov/gni/test/cntr.c

```diff
@@ -98,6 +98,7 @@ static inline void cntr_setup_eps(void)
 	hints->mode = ~0;
 
 	hints->fabric_attr->name = strdup("gni");
+	hints->caps = GNIX_EP_RDM_PRIMARY_CAPS;
```

Comment:
Can you add a comment as to why it's there.  Others should not copy this unless needed.